### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.po,./po`

### CODE REVIEW

File: src/Widgets/SearchBar.vala [hunk 1]
```diff
@@ -298,7 +298,7 @@ namespace Scratch.Widgets {
             search_context.highlight = false;
 
             if (text_buffer == null || text_buffer.text == "" || search_string == "") {
-                debug ("Can't search anything in an inexistant buffer and/or without anything to search.");
+                debug ("Can't search anything in an inexistent buffer and/or without anything to search.");
```
Comment (jeremypw) [inline src/Widgets/SearchBar.vala:301]: Should actually be `non-existent`

**Review by jeremypw — CHANGES_REQUESTED** [review]
Thanks!  Just one thing the tool corrected in the wrong way.

File: src/Widgets/SearchBar.vala (hunk 1 above)
Comment (luzpaz) [reply src/Widgets/SearchBar.vala]: @jeremypw fixed in d651bdd

File: src/Widgets/SearchBar.vala [hunk 2]
```diff
@@ -298,7 +298,7 @@ namespace Scratch.Widgets {
             search_context.highlight = false;
 
             if (text_buffer == null || text_buffer.text == "" || search_string == "") {
-                debug ("Can't search anything in an inexistant buffer and/or without anything to search.");
+                debug ("Can't search anything in an non-existent buffer and/or without anything to search.");
```
Comment (jeremypw) [inline src/Widgets/SearchBar.vala:301]: Sorry I should have said that the article will need to be changed as well.
```suggestion
                debug ("Can't search anything in a non-existent buffer and/or without anything to search.");
```

**Review by jeremypw — APPROVED** [review]

**Pull request merged**